tools/autograd/derivatives.yaml

```diff
@@ -446,6 +446,10 @@
   self: grad
   result: auto_linear
 
+- name: _simulate_lazy_clone(Tensor self) -> Tensor
```

Comment:
A bit curious what this operator does lol

Reply:
It makes a view of the input, but the view is given a new StorageImpl, and the shared data pointer is given a COWSimDeleterContext.

But I decided to remove this in favor of making `torch._lazy_clone()` switch between simulated COW and actual COW according to the `get_future_lazy_clone()` setting